rancher/charts
# [v2.8] Bump rancher monitoring rancher/shell to 0.1.22

## Issue: 


## Problem

There is a high severity curl [CVE-2023-38545](https://curl.se/docs/CVE-2023-38545.html) that was released on October 11th, it is a SOCKS5 heap buffer overflow in the rancher shell versions used by the charts for monitoring.

## Testing

Verify the helm charts deploy correctly and the associated pods are running. Then verify the uninstall works successfully.

## Engineering Testing
### Manual Testing

Verified the helm charts deploy correctly and the associated pods are running.  Then verify the uninstall works successfully.

### Automated Testing

N/A

## QA Testing Considerations

- Install, uninstall & upgrade charts

### Regressions Considerations

N/A

## Backporting considerations

N/A
